### ISSUE TITLE
Agri-gaia pattern visualization not working

### ISSUE THREAD

**Issue by miguelriemoliveira** [lardemua/atom#526]
![image](https://user-images.githubusercontent.com/11350998/183247087-5a6a871a-fe01-453b-a0eb-4be4feeb23f3.png)

**Comment by miguelriemoliveira** [comment]
I confirm the axis of the dae file  is in the center of the pattern. It should be by convention on the center of the bottom left corner

![image](https://user-images.githubusercontent.com/11350998/183247931-3664374a-a03f-4bc1-b4ae-60bbcea1d56b.png)
.

**Comment by miguelriemoliveira** [comment]
So I did some experiments. I am able to change the position of the origin in meshlab, which is my usual procedure.

The problem is that I loose the textures when I do this. 

Can you provide the pattern 3d textured mesh in another format such as obj or stl?

**Comment by miguelriemoliveira** [comment]
So I solved it creating a new chessboard pattern with the dimensions you gave me.

The convention is not as I said origin bottom left corner, x left to right, y bottom to top. That is the convention for charucos.

For chessboards the convention is origin top left corner, x left to right, y top to bottom.

Here's a visualization of the pattern during calibration

![image](https://user-images.githubusercontent.com/11350998/183290734-dbf6a958-34c8-42a6-a2fc-e37d47ef94ce.png)

I added the new pattern on the atom_calibration package so it can serve others as well.

Its in https://github.com/lardemua/atom/tree/noetic-devel/atom_calibration/meshes/chessboard_7x11_corners

Also changed agri-gaia calibration.yaml to point to this new pattern.dae

Finally,  manually edited the dataset to point to this new mesh. Here's the updated dataset:

[test1.zip](https://github.com/lardemua/atom/files/9276784/test1.zip)

when a new dataset is collected this does not have to be done of course. This was just a recycling of the old dataset.
